### ISSUE TITLE
Creatives created should have 1x1 Target ad unit size and size overrides with the actual sizes

### ISSUE THREAD

**Issue by stygianmessiah** [prebid/line-item-manager#1]
* line-item-manager version: 0.1.8

### Description

Referring to documentation [here](https://docs.prebid.org/adops/step-by-step.html) -  or [here](https://docs.prebid.org/adops/send-all-bids-adops.html), Prebid recommends creating creatives of target ad size 1x1 and having size overrides for all the sizes in each 1x1 creative. 

Prebid creatives being size agnostic, the same creatives can be used for any bid and any size that has hit the line item. This is especially useful when more than 1 bid from the page for the same size ends up hitting the same line item. GAM does not have the ability to serve the same creative twice for one size, so it will just ignore all the requests beyond the first one if they are all for the same size.

 
### What the tool currently creates
![image](https://user-images.githubusercontent.com/35668134/101801879-d1159200-3adc-11eb-81bf-2db4180d36ad.png)

### What the tool should do
![image](https://user-images.githubusercontent.com/35668134/101802147-1f2a9580-3add-11eb-9d05-301803115555.png)

Also, the number of 1x1 creatives should be equal to the number of sizes that the line items cater to, with a max limit of five. 

So if the user wants to create line items for five sizes, there should be five 1x1 creatives each with ALL five sizes in the size overrides field. 

If the user wants to create line items for seven sizes, there should still be five 1x1 creatives each with ALL seven sizes in the size overrides field.

**Comment by dshore** [comment]
```

**Comment by dshore** [comment]
As per Prebid documentation I have in development the below to address this issue, adding the "size_override" boolean defaulted to True for banner creatives:
```
###############################################################################
# Creatives (required)
# Allowed Templating Key Words: Bidder, Media, Misc
#
# For additional explanations on settings see the following:
#   banner: https://docs.prebid.org/adops/step-by-step.html
#   video: https://docs.prebid.org/adops/setting-up-prebid-video-in-dfp.html
###############################################################################
creative: # at least one of the following types is required {video, banner}
  name: "Prebid {{ bidder_name }}-{{ media_type }}"
  banner:
    # safe_frame: False # optional: defaults to True
    # size_override: False # optional: defaults to True with a 1x1 creative
    sizes: # list
      - height: 480
        width: 640
    snippet: |
      <script src = "https://..."></script>
        <script>
          ...
        </script>
```

**Comment by dshore** [comment]
This has been resolved in 0.1.10:
```
pip install --upgrade --extra-index-url https://pypi.org/simple -i https://test.pypi.org/simple/ line-item-manager
```
